core/src/cs/eng1/piazzapanic/PlayerState.java

```diff
@@ -1,32 +1,59 @@
 package cs.eng1.piazzapanic;
 
+import cs.eng1.piazzapanic.utility.Timer;
+
 public class PlayerState {
 
-    public boolean customerHappy;
-    private float happinessMult = 1.3f;
+    private final float happinessMult = 1.3f;
+
+    private static PlayerState instance = null;
+
+    public float cash = 0;
+
+    private Timer[] powerUpTimers = {
+        new Timer(10000, true, false), // doubleChefSpeed
+        new Timer(60000, false, false), // doublePrepSpeed
+        new Timer(60000, false, false), // noFailPrep
+        new Timer(60000, false, false), // noRepLoss
+        new Timer(60000, false, false), // moreMoney
+    };
+
+    public static PlayerState getInstance() {
```

Comment:
Would the Singleton pattern here mean that PlayerState can carry over multiple games? Like if I exit scenario mode and then enter endless?

Reply:
I can that the PlayerState is reset.